# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1214510774 14400
#      Thu Jun 26 16:06:14 2008 -0400
# Node ID d56511a0288475622cb5b76725cafdbc09f000bb
# Parent  cce16b4e09702152c298fd78925ed6f4f202ec1d
token.h: Omit "class symbol_table::symbol_record" decl

diff --git a/src/token.h b/src/token.h
--- a/src/token.h
+++ b/src/token.h
@@ -21,18 +21,16 @@ along with Octave; see the file COPYING.
 
 */
 
 #if !defined (octave_token_h)
 #define octave_token_h 1
 
 #include <string>
 
-class symbol_table::symbol_record;
-
 class
 token
 {
 public:
 
   enum token_type
     {
       generic_token,
